.github/workflows/cross.yaml

```diff
@@ -72,3 +72,52 @@ jobs:
             cosign-*
             cosign.-*sha256
             cosign-*.sig
+
+ build-sget:
```

Comment:
need extra space, the indentation does not match

Reply:
Oops, sorry I missed that.